environment.py

```diff
@@ -201,6 +201,10 @@ def env() -> Mapping[str, Optional[str]]:
         'azul_docker_registry': '{AZUL_AWS_ACCOUNT_ID}.dkr.ecr.'
                                 '{AWS_DEFAULT_REGION}.amazonaws.com/',
 
+        # The tag version of Azul registry's PyCharm docker image.
```

Comment:
```suggestion
        # The tag of the PyCharm image that we use to format the source code
        # in continuous integration. We specify here instead of directly in the Makefile
        # so that a change to the version does not require updating requirements.
```